05.wc/wc.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options, filenames = parse_options
+  file_details = collect_file_stats(filenames)
+  total_stats = calculate_total_stats(file_details)
```

Comment:
detailsとstatsだと名前だけ見ても何がどう違うのかイメージしづらいですね。
前回のコメントは、`[{...}, {...}, {...}]`と`{...}`があるなら、`[stat, stat, stat]`と`stat`というふうに捉えて、

```suggestion
  file_stats = collect_file_stats(filenames)
  total_stat = calculate_total_stat(file_details)
```

というように単数形と複数形を使い分けたらよいのでは？という意図でした。
（ハッシュは複数形になるときもあれば単数形になるときもあります）